assets/src/edit-story/elements/video/controls.js

```diff
@@ -78,16 +77,27 @@ const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
   }
 `;
 
-const Play = styled(PlayIcon)`
+const ButtonBackground = styled.div`
+  width: ${PLAY_BUTTON_SIZE / 2}px;
+  height: ${PLAY_BUTTON_SIZE / 2}px;
```

Comment:
The clickable area for the icon is really big, which means that even if I click outside what appears to be the button, I might still trigger play rather than drag:

![play-stop-clickable-bug](https://user-images.githubusercontent.com/637548/112231355-f00fde00-8c0c-11eb-8047-bb20af287eea.gif)

Reply:
Should be fixed now, PTAL.